package.json

```diff
@@ -3,8 +3,13 @@
   "version": "1.0.0",
   "description": "Create reusable Tailwind CSS components with React.",
   "type": "module",
-  "types": "./dist/index.d.ts",
-  "exports": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "module": "./dist/index.mjs",
```

Comment:
Never seen a `"module"` key there, do you know why it is used for? According to [conditional exports spec](https://nodejs.org/docs/latest-v20.x/api/packages.html#conditional-exports), valid values seem to be `node-addons`, `node`, `import` and `require`.

Reply:
I don't want to provide a cjs alternative. I don't know @gabrielelpidio why did you add "module" entry? Can I remove it?

Reply:
I based it off [SWR package.json](https://github.com/vercel/swr/blob/main/package.json) it might be safe to remove though